odoo/addons/base/views/res_bank_views.xml

```diff
@@ -94,6 +94,9 @@
                         </group>
                     </group>
                 </sheet>
+                <footer replace="0">
+                    <button name="action_archive_bank" type="object" string="Archive" class="btn btn-secondary"/>
```

Comment:
why a new method and not simply use `action_archive`?

Reply:
If i don't do a reload of the page, the value stays in the list view since there is not rerender